rusk-wallet/Cargo.toml

```diff
@@ -0,0 +1,34 @@
+[package]
+name = "rusk-wallet"
+version = "0.1.1"
+edition = "2021"
+
+[dependencies]
+clap = { version = "3.0", features = ["derive"] }
+tokio = { version = "1.15", features = ["full"] }
+block-modes = "0.8"
+tiny-bip39 = "0.8"
+requestty = "0.3"
+crypto = "0.3"
+whoami = "1.2"
+blake3 = "1.3"
+tonic = "0.6"
+prost = "0.9"
+dirs = "4.0"
+bs58 = "0.4"
+rand = "0.8"
+aes = "0.7"
+
+dusk-wallet-core = "0.8.0-rc.1"
```

Comment:
This should be `0.8.0-rc`. In the meantime I'll yank `rc.1` to prevent its usage.

Reply:
If you're serializing transactions this should now be `to_var_bytes` instead of `to_bytes`